tests/migrate-timestamps.test.ts

```
@@ -0,0 +1,365 @@
+    input: any;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    constructor(input: any) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      this.input = input;
+    }
+  },
+}));
+
+const { migrateTimestampsHandler } = await import(
+  "@/api/v2/assets/handlers/migrate-timestamps"
+);
```

ESLint

[error] Replace `·await·import(⏎··"@/api/v2/assets/handlers/migrate-timestamps"⏎` with `⏎··await·import("@/api/v2/assets/handlers/migrate-timestamps"`

(prettier/prettier)